src/EventListener/ExceptionListener.php

```diff
@@ -37,4 +55,35 @@ public function onKernelException(GetResponseForExceptionEvent $event)
 
         parent::onKernelException($event);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function logException(\Exception $exception, $message)
+    {
+        if (null === $this->logger) {
+            parent::logException($exception, $message);
+
+            return;
+        }
+
+        $exceptionClass = get_class($exception);
+        $statusCode = null;
+
+        foreach ($this->exceptionToStatus as $class => $status) {
+            if (is_a($exceptionClass, $class, true)) {
+                $statusCode = $status;
+
+                break;
+            }
+        }
+
+        if (null === $statusCode || $statusCode >= 500) {
```

Comment:
WDYT about making the error level that bubbles configurable?

Reply:
Btw, I think that the current behavior should be preserved when `APP_DEBUG` is set to true.

Reply:
This behavior is aligned with how symfony deals with exceptions that map to specific HTTP status codes. see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php#L88
also the level is still "error", which should be logged in dev **and** prod anyway. the error log level is used for "expected" expections/errors resulting from bad usage (at least that is my interpretation), therefore you may want to log it (to identifiy "bad" users), but you dont want to escalate it (via email, SMS) like critical and above (which denote app errors, that are not expected).

So for my part i dont think, that adjusting the behavior via `APP_DEBUG` is needed. but it could be easily dont in the service config of the listener (via expression not passing the `$exceptionToStatus` array when `APP_DEBUG` is set).

> WDYT about making the error level that bubbles configurable?

I think that this is beyond the scope of this PR, and i would rather use custom listeners for these cases. the reason for that is, that switching log level by class is very coarse grained and what you dont want, is that uncaught exception "slip" through, because the class happens to be configured with "warning" or "info" from one of the exceptions sources, but in another throwing part it is much more severe.